dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/viewtools/content/ContentMapTest.java

```diff
@@ -122,4 +139,143 @@ public void testGet_showCategories_AsAnonUser() throws DotDataException, DotSecu
+                    parentContentType.id(),
+                    childContentType.variable(), String.valueOf(
+                            RELATIONSHIP_CARDINALITY.ONE_TO_MANY.ordinal()));
+
+            //Create Contentlet
+            final ContentletDataGen parentContentletDataGen = new ContentletDataGen(
+                    parentContentType.id());
+            final ContentletDataGen childContentletDataGen = new ContentletDataGen(
+                    childContentType.id());
+
+            Contentlet contentletParent = parentContentletDataGen
+                    .setProperty(textFieldString, "parent Contentlet").next();
+
+            //Children in English
+            Contentlet contentletChild = childContentletDataGen
+                    .setProperty(textFieldString, "child Contentlet").nextPersisted();
+            contentletChild = ContentletDataGen.publish(contentletChild);
+
+            Contentlet contentletChild2 = childContentletDataGen
+                    .setProperty(textFieldString, "child Contentlet 2").nextPersisted();
+            contentletChild2 = ContentletDataGen.publish(contentletChild2);
+
+            //Children in Spanish
+            Contentlet contentletChild3 = childContentletDataGen
+                    .setProperty(textFieldString, "child Contentlet 3")
+                    .setProperty(Contentlet.LANGUAGEID_KEY,
+                            TestDataUtils.getSpanishLanguage().getId()).nextPersisted();
+            contentletChild3 = ContentletDataGen.publish(contentletChild3);
+
+            final Relationship relationship = relationshipAPI
+                    .getRelationshipFromField(relationshipField, user);
+
+            //Relate contentlets
+            final ContentletRelationships contentletRelationships = new ContentletRelationships(
+                    contentletParent);
+
+            final ContentletRelationships.ContentletRelationshipRecords contentletRelationshipRecords = contentletRelationships.new ContentletRelationshipRecords(
+                    relationship, true);
+            contentletRelationshipRecords.setRecords(
+                    CollectionsUtils.list(contentletChild, contentletChild2, contentletChild3));
+            contentletRelationships.getRelationshipsRecords().add(contentletRelationshipRecords);
+
+            //Checkin of the parent to validate Relationships
+            contentletParent = contentletAPI
+                    .checkin(contentletParent, contentletRelationships, null, null, user, false);
+            contentletParent = ContentletDataGen.publish(contentletParent);
+
+            final Context velocityContext = mock(Context.class);
+
+            final ContentMap contentMap = new ContentMap(contentletParent,
+                    userAPI.getAnonymousUser(),
+                    PageMode.LIVE, defaultHost, velocityContext);
+
+            List<ContentMap> result = (List) contentMap.get(relationshipField.variable());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'result' could be declared final](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4599592)